packages/SwingSet/src/liveslots/virtualReferences.js

```diff
@@ -160,6 +165,46 @@ export function makeVirtualReferenceManager(
     kindInfoTable.set(`${kindID}`, { reanimator, deleter, durable });
   }
 
+  /**
+   * Compare two arrays (shallowly) for equality.
+   *
+   * @template T
+   * @param {T[]} a1
+   * @param {T[]} a2
+   * @returns {boolean}
+   */
+  function arrayEquals(a1, a2) {
```

Comment:
Hey, it is a new function.

```suggestion
  const arrayEquals = (a1, a2) => {
```